pkg/directorysync/client.go

```diff
@@ -421,6 +421,12 @@ type Directory struct {
 
 	// Identifier for the Directory's Organization.
 	OrganizationID string `json:"organization_id"`
+
+	//Created At
```

Comment:
Could we make these doc comments look more like this?

- Full sentences describing the field
- Space between the `//` and the first word of the comment

```suggestion
	// The timestamp of when the Directory was created.
```

Reply:
Done!